### PR TITLE
Near 38 add aws ivs client wrapper

### PR DESCRIPTION
**Pull request by ji-min0**
## ✅ PR 한줄 요약
- aws ivs client wrapper 작성 및 유닛테스트

## 🔗 관련 이슈
- Jira: NEAR-38

## 🛠️ 변경 내용
- [x] Channel CRUD
- [x] Stream key 관리
- [x] Live stream 상태 모니터링
- [x] Stream session 분석

## 🧪 테스트
- [x] 유닛테스트 
  - [x] Channel CRUD operations (9 tests)
  - [x] Stream key management (5 tests)
  - [x] Stream health and control (7 tests)
  - [x] Session analytics (3 tests)
  - [x] Edge cases (2 tests)

### CODE REVIEW

File: app/integrations/aws_ivs/client.py [hunk 1]
```diff
@@ -1,10 +1,264 @@
...
+        """
+        [생성] IVS 채널 생성 API 호출
+        """
+
+        params: CreateChannelRequestTypeDef = {
+            "name": name,
+            "latencyMode": latency_mode,
+            "type": channel_type,
+            "authorized": authorized,
+        }
```
Comment (Meoyoug) [inline app/integrations/aws_ivs/client.py]:
```suggestion
        params = CreateChannelRequestTypeDef(
            name=name,
            latencyMode=latency_mode,
            type=channel_type,
            authorized=authorized,
        )
```

Comment (Meoyoug) [reply app/integrations/aws_ivs/client.py]: TypedDict의 장점을 살려보면 좋을것 같습니다

File: app/integrations/aws_ivs/client.py [hunk 2]
```diff
@@ -1,10 +1,264 @@
...
+
+        if recording_config_arn:
+            params["recordingConfigurationArn"] = recording_config_arn
+
+        if preset:
+            params["preset"] = cast("Any", preset)
```
Comment (Meoyoug) [inline app/integrations/aws_ivs/client.py]: 얘는 왜 Any로 캐스팅한건가용..?

Comment (Meoyoug) [reply app/integrations/aws_ivs/client.py]: preset에 대해서도 Literal을 활용해서 정확한 타입을 맞춰서 선언해주면 좋을 것 같아요
stubs에 타입정의가 어떻게 되어있는 지 확인해보면 알 수 있을 것 같습니다

File: app/integrations/aws_ivs/client.py
Comment (Meoyoug) [inline app/integrations/aws_ivs/client.py:1]: logger를 활용해서 client 예외발생 시 로깅 처리할 수 있는 래퍼함수를 만들어서 사용하는것도 좋아보이네요

Comment (Meoyoug) [inline app/integrations/aws_ivs/client.py:1]: **filters keyword argument로 필터링할 때 받을 수 있는 인자에 대해서도 타입을 명확히 해주면 좋을 것 같습니다 어떤 키를 통해 필터링이 가능케할것인지?

**Review by Meoyoug — CHANGES_REQUESTED** [review]

**Review by Meoyoug — APPROVED** [review]
